src/views/Statistic/Statistic.js

```diff
@@ -11,7 +10,7 @@ import StatisticValue from './StatisticValue'
 
 export default class Statistic extends Component {
   static propTypes = {
-    children: customPropTypes.ofComponentTypes(['Statistics', 'Label', 'Value']),
+    children: PropTypes.node,
```

Comment:
A Statistic can render more than just statistic parts.  It is not the documented use but it handles it OK.